src/std/fs.ab

```diff
@@ -76,3 +78,45 @@ pub fun change_owner(user: Text, path: Text): Bool {
 
     return false
 }
+
+/// Finds all files or directories matching a file glob.  Escapes all
+/// characters in the passed-in glob apart from "*", "?" and "/", to
+/// prevent injection attacks.
+pub fun glob(path: Text): [Text]? {
+    path = replace_regex(path, "\([^*?/]\)", "\\\\\1")
+    let files = $eval "for file in {path}; do [ -e \\\"\\\$file\\\" ] && echo \\\"\\\$file\\\"; done"$?
+    return split(files, "\n")
+}
+
+/// Finds all files or directories matching multiple file globs.  When
+/// we have union types, these two functions can be merged as follows:
```

Comment:
The code of the hypothetical union version shouldn't be in the end user documentation, please remove it. Instead, simply explain that this function will be removed in favor of the regular one once we have union types.

Reply:
Fair enough, done.